monero-outreach-docs/translations/pt-br/storyboard/blockchain_gender_pt_br.md

```diff
@@ -0,0 +1,78 @@
+
+Eu só posso falar por mim mesma, mas sinto que os homens com quem tenho me comunicado no espaço Cripto/Monero foram muito receptivos. Felizmente, não senti nenhuma reação negativa por ser mulher. Fui capaz de me aproximar de pessoas que trabalham no Monero sem nenhum problema até agora, ter acesso às mesmas informações e trabalhar em qualquer projeto, assim como qualquer outra pessoa.
+
+Se você estiver interessado e quiser fazer parte do espaço Cripto de alguma forma, vá em frente. É dessa forma que você supera as barreiras, fazendo parte dele e focando no principal: em como você pode fazer a diferença e melhorar seu projeto de interesse.
+
+**Algumas mulheres, como [Amber Baldet e Elizabeth Stark](https://qz.com/1262167/the-first-rule-of-being-a-woman-in-crypto-is-you-do-not-Talk-about-being-a-woman-in-crypto/), estão relutantes em usar o rótulo de “mulheres no blockchain” e evitam realçar a disparidade de gênero. Elas argumentam que é contra intuitivo seus esforços serem reconhecidos por suas contribuições, ao invés de primariamente pelo seu gênero. Além disso, argumentam que o foco na disparidade de gênero desestimulará a participação de mais mulheres e, na verdade, a maneira mais eficaz de aumentar o número de mulheres nessa indústria é chamando a atenção para as mulheres que são bem-sucedidas. Qual é a sua opinião sobre isso?**
+
+Eu concordo completamente com elas. Precisamos nos concentrar e escrever sobre as conquistas das mulheres, em vez de apontar a disparidade de gêneros. Neste espaço, sinto que você é reconhecido pelo que você contribui. É sobre avançar com um objetivo em mente, a descentralização, não importa com qual projeto você se identifica mais.
+
+Eu apoio evitar rotulações e discussões a respeito de gênero e, ao invés disso, focar em encontrar mulheres que estão contribuindo neste espaço e certificando-se de que seu trabalho está sendo exposto. Há muitas mulheres por aí interessadas em criptografia.
+
+**Você acha que existem situações em que o projeto ou a comunidade Monero são particularmente propícios para atenuar a disparidade de gênero na indústria de blockchain?**
+
+Eles não discriminam - se você tem uma ideia que parece viável, você receberá apoio. Mais uma vez, eu só posso falar por mim mesma, mas na minha experiência, senti que minhas ideias e sugestões foram levadas a sério pelas pessoas com quem lidei. Os homens no espaço Monero/Cripto foram acolhedores e solidários.
+
+É claro que você será exposto a trolls e sexismo (isso acontece em todos os ambientes), mas isso tem a ver com o caráter inseguro do indivíduo e não deve ser uma generalização de todos os homens. Na maioria das vezes, se a ideia é boa e melhora a tecnologia ou ajuda a comunidade de alguma forma, ela é apoiada - não importa quem tenha a ideia.

```

Comment:
Change gender: "será exposto" to "será expost**a**".

Reply:
Evaluate if we can change "não importa quem tenha a ideia." to "não importa quem a tenha dado".